v1.0.0.yaml

```diff
@@ -2600,94 +2342,179 @@ paths:
-                    Model: Canon EOS 40D
-                    Orientation: 1
-                    XResolution: 72
-                    YResolution: 72
-                    ResolutionUnit: 2
-                    Software: GIMP 2.4.5
-                    ModifyDate: 2008:07:31 10:38:11
-                    YCbCrPositioning: 2
-                    ExifOffset: 214
-                    GPSInfo: 978
-                  thumbnail:
-                    Compression: 6
-                    XResolution: 72
-                    YResolution: 72
-                    ResolutionUnit: 2
-                    ThumbnailOffset: 1090
-                    ThumbnailLength: 1378
-                  exif:
-                    ExposureTime: 0.00625
-                    FNumber: 7.1
-                    ExposureProgram: 1
-                    ISO: 100
-                    ExifVersion: "0221"
-                    DateTimeOriginal: 2008:05:30 15:56:01
-                    CreateDate: 2008:05:30 15:56:01
-                    ShutterSpeedValue: 7.375
-                    ApertureValue: 5.625
-                    ExposureCompensation: 0
-                    MeteringMode: 5
-                    Flash: 9
-                    FocalLength: 135
-                    SubSecTime: "00"
-                    SubSecTimeOriginal: "00"
-                    SubSecTimeDigitized: "00"
-                    FlashpixVersion: "0100"
-                    ColorSpace: 1
-                    ExifImageWidth: 100
-                    ExifImageHeight: 68
-                    InteropOffset: 948
-                    FocalPlaneXResolution: 4438.356164383562
-                    FocalPlaneYResolution: 4445.969125214408
-                    FocalPlaneResolutionUnit: 2
-                    CustomRendered: 0
-                    ExposureMode: 1
-                    WhiteBalance: 0
-                    SceneCaptureType: 0
-                  gps:
-                    GPSVersionID:
-                      - 2
-                      - 2
-                      - 0
-                      - 0
-                  interoperability:
-                    InteropIndex: R98
-                    InteropVersion: "0100"
-                  makernote: {}
-        "401":
-          description: Unauthorized
+                properties:
+                  message:
+                    type: string
+                    example: The requested file does not exist or is inaccessible at this time.
+                  help:
+                    type: string
+                    example: For support kindly contact us at support@imagekit.io.
+        "400":
+          description: Bad request.
           content:
             application/json:
               schema:
                 type: object
                 properties:
                   message:
                     type: string
-                    example: Your request does not contain private API key.
+                    example: The requested file is corrupt or invalid.
```

Comment:
To add:
- `"Your requested URL is not a valid ImageKit URL."`

Reply:
i have added this in get metadata by url example. will be confusing in this one as user is not passing url, instead user is passing file id so they won't get this error.